### PR TITLE
Join Class of 2020

### PR DESCRIPTION
**Pull request by CRefice**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] ~~**Tier 2** ✏️: I wrote a post on DEV about a project I built.~~
  - [ ] ~~**Tier 3** ✨: I want my project to be highlighted on stream.~~
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/CRefice/CRefice.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Carlo Refice
+institution: University of Groningen
+profile_pic: CRefice.jpg
+quote: "Programming is like a cup of morning coffee: it's comforting, stimulating, and it causes addiction."
```
Comment (Rishit-dagli) [inline _data/CRefice/CRefice.md:5]: Please do not include quotes(")

**Comment by CRefice** [comment]
This is a duplicate of #3090, which was not merged due to wrong formatting. I double checked that I don't use any ASCII characters, the only thing different from the example md file given was double quotes around my senior quote.  

Should I fill in the swag shipment form again by linking this PR instead?

File: _data/CRefice/CRefice.md (hunk 1 above)
Comment (CRefice) [reply _data/CRefice/CRefice.md]: Done, though the example MonaTheOctocat .md file does use them.

**Comment by Rishit-dagli** [comment]
I am not a person behind the GitHub Education team but as far as I know, since #3090 has been closed you should refill the form with this PR  link.

**Comment by CRefice** [comment]
> We have reviewed 📝 your application to the yearbook. It looks great ✨ but we found that some headers might be missing in the Markdown file or the format is broken (used any non-ascii characters?). I will close this PR so you can do the changes requested. You can open a new one once you've done the fixes and we will review it again.😊

I double checked the Markdown file for any format differences or non-ascii characters, but I couldn't find any. In fact, this is the second PR I opened. The reviewer made sure that all of the checkboxes are ticked, so I don't understand what is happening...

**Comment by elisemoe** [comment]
Hi @CRefice 👋 It looks like the “:” in your quote is what’s being flagged. Please update your quote and we’ll review again 😊

**Pull request closed without merging**